OneCommunityGlobal/HGNRest
# (PRIORITY URGENT) Jae: Some (but not all) tasks are not saving my updates from the Tasks page backend

3/17 update: This has been an ongoing problem for me. At one point I thought it was just my computer but I’ve just replicated this on my computer, my wife’s computer and my phone.
3/21 update: It’s not the tasks that’s causing this problem, I can go in later on a task and it will sometimes work. Phone seems to work more often than computer for the same task, but today I tried to edit one on my phone and it didn’t work and then I was able to edit it on my computer.
Here's a video I shot just now of this: https://www.dropbox.com/s/zrqmfqbtpnbr6ey/Tasks%20not%20updating.mov?dl=0

### **Main Changes Explained:**
I added category to the taskschema. 

### **How to Test:**

1. check out this branch and the frontend branch that is related to this PR as well. (see link below)
2. select a task that has been assigned to a user. This will send you to the page for that task.
3. select edit and a small window should appear and allow you to make changes to that task.
4. make changes to each section and click on the update button
5. the page should reload and the new data should render.

### **Additional Notes:**
This branch needs the frontend branch https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/756
